client/src/app/components/subcomponents/edit-buttons/edit-buttons.component.html

```diff
@@ -0,0 +1,2 @@
+<button class="btn btn-secondary" [routerLink]="['/note', JSON.stringify(note)]"> Edit </button>
+<button class="btn btn-danger" [routerLink]="['/note/destroy', note.id]"> Delete </button>
```

Comment:
Instead of redirecting to "/note/destroy" you can create a function "destroy()", that accepts "note.id" as parameter, that deletes it directly.